llvm/docs/GlobalISel/GenericOpcode.rst

```diff
@@ -639,6 +639,11 @@ Concatenate two vectors and shuffle the elements according to the mask operand.
 The mask operand should be an IR Constant which exactly matches the
 corresponding mask for the IR shufflevector instruction.
 
+G_SPLAT_VECTOR
+^^^^^^^^^^^^^^^^
+
+Create a vector where all elements are the scalar from the source operand.
+
```

Comment:
Unless we do the same in the DAG, I think this just adds complication down the line for pattern sharing. The buildBuildSplatVector can just create the G_BUILD_VECTOR with N copies of the input